kafka_exporter.go

```diff
@@ -33,6 +33,13 @@ const (
 	clientID  = "kafka_exporter"
 )
 
+const (
+	OFF = 0
```

Comment:
what this OFF = 0 is? When on `glog` you don't use the verbosity function, it just prints the message provided to `Info` function so I would use INFO = 0 as default and DEBUG = 1 and TRACE = 2.

Reply:
I thought it would be good to have a possibility to disable info logging.